Dockerfile

```diff
@@ -1,20 +1,21 @@
-FROM ruby:2.6.0
+FROM ruby:2.6.4-alpine
```

Comment:
Alpine is a minimal size linux distro.  Prior to this PR, the image size of each image (multiply sizes by 3) was 1938MB.  By using this build process instead using `FROM ruby:2.6.4` it bring it down to 963MB.  By switching to alpine we bring it all the way down to 308MB.